model/vm_host.rb

```diff
@@ -184,7 +184,7 @@ def install_rhizome(install_specs: false)
   end
 
   # Introduced for downloading a new boot image via REPL.
-  def download_boot_image(image_name, version:, custom_url: nil)
+  def download_boot_image(image_name, version: nil, custom_url: nil)
```

Comment:
I prefer to not have a default for `version` & pass `nil` explicitly when we intend to, to reduce operator error of unintended unversioned image downloads.

Reply:
I did `version` mandatory again. As we can still set `version: nil`, and might need to download unversioned images, I removed `default_version = Config.send(:"#{image_name.tr("-", "_")}_version")` helper. It overwrites `nil` version anyway even we want download unversioned one. We can add it when we decide to disallow unversioned images. For now, the operator can run download image method as `vm_host.download_image("github-ubuntu-2204", Config.github_ubuntu_2204_version)`